### PR TITLE
Milestone 1: React project - Calculator

### PR DESCRIPTION
**Pull request by fivan18**
## Project specific requirements
- [ ] Use create-react-app to initialize a new React app
- [ ] Delete any unnecessary file or asset
- [ ] Add prop-types using npm for typechecking props
- [ ] Add big.js using npm
- [ ] Run the application with npm start and make sure that everything works correctly
- [ ] Follow the instructions in create-react-app-buildpack to deploy the app to Heroku

### CODE REVIEW

File: src/app/App.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        {' '}
+        {rest.toString()}
+      </div>
+    </div>
+  );
+}
+
+App.propTypes = {
+  enterString: PropTypes.string.isRequired,
+};
```
Comment (bellom) [inline src/app/App.js:24]: Hi, I am kinda curious do you really need this?

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @fivan18  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly update your `README` file and remove unnecessary content**.
Checkout out the below link for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/app/App.js (hunk 1 above)
Comment (fivan18) [reply src/app/App.js:24]: Hi @bellom, 
No actually. I just checked `Big` and `PropTypes` to work fine. All will be removed in the subsequent milestones.

Thanks for the review.

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!


Good luck with your next milestone. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Pull request merged**